lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -69,3 +69,5 @@ ecdsa==0.13
 # Flexible BAM index naming
 pysam==0.8.4+gx1
 
+# CSG dependencies
+numpy-stl==1.8.0
```

Comment:
Thomas for generating a wheel that is automatically shipped with Galaxy you need to create a wheel on our build-server with https://github.com/galaxyproject/starforge

Example PR is for example here: https://github.com/galaxyproject/starforge/pull/100